README.md

```diff
@@ -199,7 +199,8 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Bootstrap v3.3.7](http://trbootstrap.com/)
 - [CSS Stil Rehberi](https://github.com/omergulcicek/css-stil-rehberi) - [Ömer Gülçiçek](https://twitter.com/omergulcicek)
 - [CSS Grid Layout Rehberi](https://github.com/ahmetozantekin/css-grid-layout-rehberi)
-
+- [Sass Rehberi](https://github.com/dferhat/sass-rehberi) - [Ferhat Demir](https://twitter.com/ddemirferhat)
```

Comment:
Biraz nitpicking gibi olacak ama Css degil CSS, SASS degil Sass, ve Scss degil SCSS :)
Hepsinin sebepleri ve tarihsel gelisimleri var. Yazi boyunca surekli degisen sekillerde kullanilmalari daha sonra hangisinin ne oldugunu hatirlamayi baya olanaksiz kiliyor.

Reply:
İlk fırsatta düzeltiyorum, teşekkürler geri bildiriminiz için 👍